crates/forge/tests/integration/spy_events.rs

```diff
@@ -384,3 +386,74 @@ fn event_emitted_wrong_data_asserted() {
         "but keys or data are different"
     );
 }
+
+#[test]
+fn use_multiple_spies() {
```

Comment:
I'd also consider verifying that in case only one contract emits events and other not,`.emitted_events()` are empty for other contracts.

So there isn't some kind of interference between them.